src/Query/Builder.php

```diff
@@ -22,6 +22,7 @@ class Builder implements BuilderInterface
     protected $inKeys = '';
     protected $numericFilters = [];
     protected $geoFilters = [];
+    protected $sortBy = [];
```

Comment:
If I am reading the doc for [SORTBY](http://redisearch.io/Sorting/#specifying-sortby) correctly, you can only have one SORTBY statement per query. That means that $sortBy should be a string, not an array.

Reply:
Fixed